File: InTouch/Materialise.InTouch.WebSite/ViewModel/FullScreenPostPartViewModel.cs
using System;
using Materialise.InTouch.BLL.Enumes;
using Materialise.InTouch.WebSite.Model;

namespace Materialise.InTouch.WebSite.ViewModel
{
    public class FullScreenPostPartViewModel
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public DateTime CreatedDate { get; set; }
        public ContentType ContentType { get; set; }
        public FileInfoViewModel File { get; set; }
        public string VideoUrl { get; set; }
        public int DurationInSeconds { get; set; }
        public string PostType { get; set; }
        public string Priority { get; set; }
    }
}

File: InTouch/Materialise.InTouch.WebSite/ClientApp/app/components/fullscreenMode/fullscreen-post-part-view-model.ts
import { FileView } from '../post/shared/fileView';

export class FullScreenPostPartViewModel {
    public id: number;
    public title: string;
    public createdDate: Date;
    public contentType: number;
    public file: FileView;
    public videoUrl: string;
    public durationInSeconds: number;
    public postType: string;
    public priority: string;
}